base/shared/src/main/scala/scalaz/data/KleisliFunctions.scala

```diff
@@ -0,0 +1,7 @@
+package scalaz
+package data
+
+trait KleisliFunctions {
+  @inline def wrap[F[_], A, B](k: A => F[B]): Kleisli[F, A, B] = Kleisli.wrap(k)
+  @inline def run[F[_], A, B](k: Kleisli[F, A, B]): A => F[B]  = Kleisli.run(k)
```

Comment:
I think this will steal `run` in `AllFunctions` for any newt (since their erasure is always `Object`).

Besides that, I feel uncomfortable overloading names there because of how tricky Scala's overloading resolution can be in the presence of eta-expansion.

Reply:
Yeah I'll rename it to `runKleisli`- no worries